src/resources/views/corporation/partials/location.blade.php

```diff
@@ -0,0 +1,11 @@
+@if (! is_null($row->location))
+  {{ $row->location->name }}
+@else
+  Unknown Location
+@endif
+
+@if (! is_null($row->ship_type_id))
+  <i class="pull-right" data-toggle="tooltip" title="" data-original-title="{{ $row->type->typeName }}">
```

Comment:
be carefull about the null type here :)

Reply:
this one i need some help with. I copied the said part from: the current blade:

https://github.com/eveseat/web/blob/master/src/resources/views/corporation/tracking.blade.php#L54-L58

Do i need to cover for a potential null pointer exception?